### ISSUE TITLE
How to set release ID?

### ISSUE THREAD

**Issue by zachwaugh** [getsentry/sentry-cocoa#262]
_Platform:_
- [x]  iOS
- [ ]  tvOS
- [ ]  MacOS
- [ ]  watchOS

_Swift:_
- [x]  Yes -> If yes, which version? 4.1
- [ ]  No

_sentry-cocoa installed with:_
- [ ] CocoaPods
- [x] Carthage
- [ ] Manually

Version of sentry-cocoa:
**4.1.0**

---
I have following issue:

How do you set the release identifier in the Cocoa SDK? The docs say to call `setRelease` function for another SDK, but I don't see anything matching that in the Cocoa version (https://docs.sentry.io/learn/releases/#tag-errors). The default release identifier provided by the SDK isn't granular enough as it doesn't include the build number in the release name.

**Comment by HazAT** [comment]
So by default it should set your bundleID,
if that is not what you want you can use

```
self.client.beforeSerializeEvent = ^(SentryEvent * _Nonnull event) {  
    event.releaseName = @"test";
};
```

hope this helps.

**Comment by zachwaugh** [comment]
Thanks! By default it sets the bundle identifier with the version appended (`com.example.app-2.0`) The main issue here is that if we have release `2.0 (build 100)` and there is a crash that we fix in `2.0 (build 101)`, Sentry will give them the same release so we can't say when it was resolved. Can you tell me if there is a reason the Cocoa SDK doesn't use the build number by default in the release id? The docs aren't very clear about Releases and it's not mentioned at all in the Cocoa SDK. I saw the code comment mentions only to change it "if you know what you're doing". We're just trying figure out the best way to use Sentry in our apps.

**Comment by HazAT** [comment]
Oh I see.
So in this case I would just set:
```
self.client.beforeSerializeEvent = ^(SentryEvent * _Nonnull event) {  
    event.releaseName = @"com.example.app-2.0.200";
};
```
or something like this.
This should fix the feature (fixed in next release).
